app/controllers/users_controller.rb

```diff
@@ -0,0 +1,17 @@
+class UsersController < ApplicationController
+  def signup
```

Comment:
can we call this action `create` of REST terminology? because we're creating a user.